kuma/users/admin.py

```diff
@@ -6,6 +6,19 @@
 from kuma.core.managers import NamespacedTaggableManager
 from .models import UserBan, UserProfile
 
+from django.contrib.auth.admin import UserAdmin
+from django.contrib.auth.models import User
+
+
+class ExtendedUserAdmin(UserAdmin):
+    # extend the admin view of users to show date_joined field; add a filter on the field too
+    list_display = ('username', 'email', 'first_name', 'last_name', 'date_joined', 'is_staff', 'is_active')
+    list_filter = ('is_staff', 'is_superuser', 'is_active', 'date_joined',)
+
```

Comment:
You may want to add a `ordering` attribute as well to define to auto sort it by `date_joined` instead of `username` (which is the default)